### DISCUSSION TITLE
[제안] 페이지 컴포넌트 제작 관련 제안

### DISCUSSION THREAD

**Discussion by DragonTiger92** [FRONTENDSCHOOL8/Book-Kong#133] [Ideas]
페이지 컴포넌트 제작 관련 제안하고 싶은 내용이 두 가지 있습니다.
 첫 번째는 프로젝트 폴더 내 페이지 컴포넌트 JSX 파일들을 해당 컴포넌트 이름이 담긴 폴더 내에서 관리하는 안건입니다. 그리고 폴더 내에 .stories.js 파일도 같이 만들어서 관리했으면 좋겠습니다. 
 두 번째 안건은 페이지 컴포넌트의 이름 뒤에 Page를 붙여서 페이지 컴포넌트라는 걸 알기 쉽게 네이밍하는 것에 대한 안건입니다. 예를 들면 '기록 페이지'의 경우 RecordPage로 컴포넌트 이름을 붙이는 식이 될 것 같습니다.
 확인해주시고, 괜찮으시다면 나중에 각자 맡은 페이지 컴포넌트 관련 PR 보내실 때 해당 내용 반영해서 부탁 드립니다. 감사합니다!

**Comment by yen815** [comment]
넹! 좋습니다

**Comment by DragonTiger92** [comment]
해당 내용 반영해서 PR 부탁 드립니다. 일단 이 discussion은 닫도록 하겠습니다!
